Packs/FireMonSecurityManager/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,8 @@
+
+#### Incident Types
+- **FireMon Pre Change Assessment**
+- **FireMon Policy Planner**
+
+#### Integrations
+##### FireMon Security Manager
+- Fixing playbook Id
```

Comment:
```suggestion
#### Incident Types
##### FireMon Policy Planner
 - Fixed Layout name.
##### FireMon Pre Change Assessment
 - Fixed Layout name.
```

Reply:
If these are mere recommendation, i would like to skip making these changes. I exactly followed what the instructions  said at the command line while running command to generate readme file. I am extremely sorry but this is just getting a bit ridiculous.